vms/administrator/services.py

```diff
@@ -0,0 +1,17 @@
+from django.core.exceptions import ObjectDoesNotExist
+from administrator.models import Administrator
+
+
+def get_administrator_by_id(admin_id):
```

Comment:
Why do you want to do all this, when you can just: 
`return  Administrator.objects.filter(pk=admin_id).first() `

Reply:
@tonythomas01 I read the current code, and saw that this was the convention that was being followed. Like in volunteer/ services.py . Should I delete this file then?